src/main/java/uk/gov/hmcts/reform/finrem/caseorchestration/controllers/CheckLatestConsentOrderController.java

```diff
@@ -41,16 +43,17 @@
         validateCaseData(callback);
 
         Map<String, Object> caseData = callback.getCaseDetails().getData();
+        long caseId = callback.getCaseDetails().getId();
```

Comment:
in prev file you changed `long` to `Long`. Why not the same type everywhere?

Reply:
Yep simple typo when re-writing it, will put it back though in new commit